packages/remix-edge-adapter/src/index.ts

```diff
@@ -1 +1 @@
-console.log('nothing for now')
+console.log('nothing for now');
```

Comment:
prettier in action here and in files below

Reply:
We should keep semi false